common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -224,18 +237,35 @@ def import_static(self, data_path, dest_id):
         """
         Import all static items into the content store.
         """
-        if self.static_content_store is not None and self.do_import_static:
-            # first pass to find everything in /static/
-            import_static_content(
-                data_path, self.static_content_store,
-                dest_id, subpath='static', verbose=self.verbose
-            )
-
-        elif self.verbose and not self.do_import_static:
+        static_content_importer = StaticContentImporter(
+            self.static_content_store,
+            course_data_path=data_path,
+            target_id=dest_id
+        )
+        if self.verbose and not self.do_import_static:
             log.debug(
                 "Skipping import of static content, "
                 "since do_import_static=%s", self.do_import_static
             )
+            if self.do_import_code_lib:
+                log.debug(
+                    "Importing code library anyway "
+                    "since do_import_code_lib=%s", self.do_import_code_lib
+                )
```

Comment:
Feels strange to have these logs separated out from the code below that's actually doing what the logs are telling the user. Should they be placed below next to those lines of code?

Reply:
Will do. Not sure why I did this